tickplate.js

```diff
@@ -1,15 +1,21 @@
 'use strict';
+const vm = require('vm');
+const fs = require('fs');
+const path = require('path');
 
-const tickplate = (strings, ...keys) => {
-  console.dir({ strings, keys });
-  return (values) => {
-    console.dir({ values });
-    const result = [strings[0]];
-    keys.forEach((key, i) => {
-      result.push(values[key], strings[i + 1]);
-    });
-    return result.join('');
-  };
+const tickplate = (template, sandbox, callback) => {
+  const cxt = Object.freeze(sandbox);
+  const file = path.resolve(__dirname, template);
```

Comment:
And even if we do render templates from files only, this line certainly contains a bug. Try to require the package from another project and see what happens.